README.md

```diff
@@ -0,0 +1,46 @@
+# SFML.Net - Simple and Fast Multimedia Library for .Net [![Build Status](https://travis-ci.org/SFML/SFML.Net.svg?branch=master)](https://travis-ci.org/SFML/SFML.Net)
```

Comment:
Only when I looked at it, I realized, that this was part of the main title. I believe this should be on its own dedicated paragraph.